packages/dso-toolkit/src/components/form/form-groups/textarea/form-group-textarea.scss

```
@@ -17,6 +17,7 @@
     &.form-control {
       block-size: auto;
 
+      /* prettier-ignore */
```

prettier wil dit op een manier formateren die stylelint laat piepen over   scss/operator-no-newline-after. DIe laatste leek mij prevaleren boven prettier, dus daarom prettier-ignore

---

```suggestion
      &:not([rows]) {
        $lines-block-size: typography.$line-height-base * css-form-variables.$textarea-default-rows * 1em;
        $lines-block-padding: (units.$padding-base-block + 1) * 2;

        block-size: calc(#{$lines-block-size} + #{$lines-block-padding});
      }
```

Wat denk je hiervan? Prettier kan z'n werk doen, ESLint heeft gelijk, én we hebben leesbare code